### ISSUE TITLE
Descrepancy between HMI and MOBILE APIs for vehicle data structures

### ISSUE THREAD

**Issue by dboltovskyi** [smartdevicelink/sdl_requirements#41]
There is a discrepancy in HMI and MOBILE APIs for some vehicle data structures on optionality of parameters: 
E.g.: In GetVehicleData response there is a parameter "gps" with type "GPSData"
In HMI API all parameters of GPSData structure are optional, but in MOBILE API all of them are mandatory.
Currently SDL is unable to transfer response from HMI to Mobile application with some missing parameters of GPSData structure.

List of structures with differences:
- GPSData
- TireStatus
- DeviceStatus

Please clarify which API is correct?

**Comment by KhrystynaDubovyk** [comment]
@robinmk, please advise on the issue which API is correct  

| HMI API (are optional)  | MOBILE API (are mandatory) |
| ------------- | ------------- |
| [GPSData](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/interfaces/HMI_API.xml#L2194)      | [GPSData](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/interfaces/MOBILE_API.xml#L1685) |
| [TireStatus](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/interfaces/HMI_API.xml#L2284)    | [TireStatus](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/interfaces/MOBILE_API.xml#L1659) |
|[DeviceStatus](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/interfaces/HMI_API.xml#L2381)|[DeviceStatus](https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/interfaces/MOBILE_API.xml#L1366)|

**Comment by robinmk** [comment]
I think they should be marked optional to not cause any compatibility issues.
@JackLivio @mrapitis @yang1070 , thoughts?

**Comment by mrapitis** [comment]
These structs should probably have been originally made optional in the mobile api to allow the hmi to send data when its available.  Moving forward I think we may need to deprecate the mandatory flags and resolve inconsistencies.

**Comment by AStasiuk** [comment]
@robinmk, @mrapitis so is it final decision to mark mentioned above parameters as an optional?

**Comment by robinmk** [comment]
We will bring this up as an item for discussion in the next SDLC meeting so that the entire SDLC is aware of the problem.

**Comment by AStasiuk** [comment]
@robinmk, any updates regarding this issue?

**Comment by AByzhynar** [comment]
@robinmk Do we have any updates?

**Comment by AByzhynar** [comment]
@robinmk Please reply : do we have any updates related to this issue?

**Comment by robinmk** [comment]
No, there is no new updates on this. I will have to follow up on this.

**Comment by KhrystynaDubovyk** [comment]
the proposal aligning TireStatus API https://github.com/smartdevicelink/sdl_evolution/blob/da73ad1c298fd5cc3517c9e0a4a368974c0a4a57/proposals/0236-TireStatus-Mismatch.md
